test/SlowTests/Client/TimeSeries/TimeSeriesAndDocumentConflicts.cs

```diff
@@ -0,0 +1,393 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FastTests.Server.Replication;
+using Raven.Client;
+using Raven.Client.Documents.Operations.TimeSeries;
+using Raven.Client.ServerWide;
+using Raven.Server.Documents;
+using Raven.Tests.Core.Utils.Entities;
+using Sparrow;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Client.TimeSeries
```

Comment:
What if we have a conflict on a time-series casing?
e.g. "Heartrate" vs "HeartRate"

Reply:
hadnled